main/core/src/mill/eval/Evaluator.scala

```diff
@@ -670,12 +670,16 @@ object Evaluator {
 
     val sortedGroups = Graph.groupAroundImportantTargets(topoSorted) {
       case t: NamedTask[Any] =>
-        val segments = t.ctx.segments
-
-        val additional =
-          if (!overriden(t)) Nil
-          else
-            Seq(Segment.Label("overriden")) ++ t.ctx.enclosing.split("\\.|#| ").map(Segment.Label)
+        val segments0 = t.ctx.segments
+        val (segments, additional) =
+          if (!overriden(t)) (segments0, Nil)
+          else {
+            val Segment.Label(tName) = segments0.value.last
+            (
+              Segments(segments0.value.reverse.tail.reverse: _*),
+              Seq(Segment.Label(tName + ".overriden")) ++ t.ctx.enclosing.split("\\.|#| ").map(Segment.Label)
```

Comment:
Shall we rename this from `.overriden` to `.override`? or `.overridden` to at least spell it correctly :P

Reply:
I renamed this one at least 3 times in the last two years, but then reverted to avoid nitpicking. Also, after looking into a dictionary, "overriden" is actually not incorrect. But now, as you see it the same, I'd glad to rename it. I think "overridden" is more correct than "override" as the folder is holding the state of "overridden" targets.